rslearn/data_sources/gcp_public_data.py

```diff
@@ -140,22 +145,24 @@ def __init__(
 
         self.index_cache_dir.mkdir(parents=True, exist_ok=True)
 
-        self.bucket = storage.Client.create_anonymous_client().bucket(self.bucket_name)
+        self.bucket = storage.Client().bucket(self.bucket_name)
```

Comment:
Oops, this change is not needed, I was running into rate limit errors and tried it to see if it would help but it didn't.

I rolled this back but also added documentation about credentials now being needed for the rtree index since it needs to be created from the BigQuery table rather than from the index.csv.gz (which isn't updated anymore).